src/RelayClient.ts

```diff
@@ -254,55 +255,36 @@ class RelayClient extends EnvelopingEventEmitter {
 
   // eslint-disable-next-line @typescript-eslint/ban-ts-comment
   // @ts-ignore FIXME: remove the ts ignore once the method is consumed
-  private _prepareRelayHttpRequest = async (
-    { feesReceiver }: HubInfo,
```

Comment:
We need to keep this to be able to build the request.

Reply:
you're right.

Reply:
done